### DISCUSSION TITLE
Общий чат

### DISCUSSION THREAD

**Discussion by DaovaJo** [DaovaJo/Isys_engineering#1] [Announcements]

**Comment by DaovaJo** [comment]
Привет, здесь можно отписываться по задачам проекта, может так будет удобней, на ваше усмотрение

**Reply by DaovaJo** [reply]
Короче всю документацию, отчетные файлы прикреплчйте в раздел Вики в репе

![image](https://user-images.githubusercontent.com/103495315/193531548-8add42fb-2693-49b3-8658-01ea4b9e8ae5.png)

**Comment by DaovaJo** [comment]
Реализация дищайна через код 

![IMG_20221110_170249_847.jpg](https://user-images.githubusercontent.com/103495315/201112228-2313386e-7279-452e-9a92-ea2d754a2149.jpg)

![IMG_20221110_170247_728.jpg](https://user-images.githubusercontent.com/103495315/201112253-a2999a33-a17b-4179-beca-d5d8eb813d80.jpg)

**Comment by Bibitriy** [comment]
[диаграммы по проекту.docx](https://github.com/DaovaJo/Isys_engineering/files/10236438/default.docx)

**Comment by Bibitriy** [comment]
[диаграммы по проекту.docx](https://github.com/DaovaJo/Isys_engineering/files/10287671/default.docx)

**Comment by Bibitriy** [comment]
[диаграммы по проекту.docx](https://github.com/DaovaJo/Isys_engineering/files/10288010/default.docx)
